src/main/java/com/fivetran/agent/mysql/source/TableDefinitions.java

```diff
@@ -25,11 +26,21 @@ public TableDefinitions(Query query) {
         Map<ColumnRef, List<ColumnAttributes>> allColumnAttributes = queryAllColumnAttributes(query);
         allColumnAttributes.forEach((columnRef, columnAttributes) -> {
             TableRef tableRef = columnRef.table;
-            tables.putIfAbsent(tableRef, new TableDefinition(tableRef, new ArrayList<>()));
+            tables.putIfAbsent(tableRef, new TableDefinition(tableRef, new ArrayList<>(), new HashMap<>()));
```

Comment:
NIT: It seems a little weird to me to have the TableDefinition constructor take in an empty ArrayList/HashMap. Why not just initialize `columns` and `foreignKeys` inside the class, then make a single parameter constructor? The Spec classes can use this one.